test-tests/shared/src/test/scala/zio/test/DefaultTestReporterSpec.scala

```diff
@@ -1,32 +1,29 @@
-package zio.test
-
-import zio._
 import zio.clock.Clock
-import zio.test.environment._
 import zio.test.Assertion._
 import zio.test.ReportingTestUtils._
-import zio.test.TestUtils.label
+import zio.test._
+import zio.test.environment.{TestClock, TestConsole, TestEnvironment, testEnvironmentManaged}
+import zio.{Cause, IO, Managed, ZIO}
 
 import scala.concurrent.Future
 
-object DefaultTestReporterSpec extends AsyncBaseSpec {
-
-  val run: List[Async[(Boolean, String)]] = List(
-    label(reportSuccess, "correctly reports a successful test"),
-    label(reportFailure, "correctly reports a failed test"),
-    label(reportError, "correctly reports an error in a test"),
-    label(reportSuite1, "correctly reports successful test suite"),
-    label(reportSuite2, "correctly reports failed test suite"),
-    label(reportSuites, "correctly reports multiple test suites"),
-    label(simpleAssertion, "correctly reports failure of simple assertion"),
-    label(multipleNestedFailures, "correctly reports multiple nested failures")
-  )
+object DefaultTestReporterSpec
+    extends ZIOBaseSpec(
+      suite("DefaultTestReporterSpec")(
+        testM("report success") {
+          assert(ZIO.effect(DefaultTestReporterSpecUtil
+            .check(test1, Vector(test1Expected, reportStats(1, 0, 0)))), isFalse)
+        }
+      )
+    )
+
+object DefaultTestReporterSpecUtil {
 
   def makeTest[L](label: L)(assertion: => TestResult): ZSpec[Any, Nothing, L, Unit] =
```

Comment:
This method isn't really doing anything. We can just replace all occurrences of `makeTest` with `test` using the existing method in the `zio.test` package object.